docs/developers/optimistic-oracle-integration.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Integrating with the Optimistic Oracle
+sidebar_label: Integrating with the Optimistic Oracle
+---
+
+## "Deposit Box" Example Contract
+
+This example will set up an `OptimisticDepositBox` contract which custodies a user’s ERC-20 token balance.
+
+On a local testnet blockchain, the user will deposit wETH (Wrapped Ether) into the contract and withdraw wETH corresponding to a desired USD amount.
+
+The user links the `OptimisticDepositBox` with one of the price identifiers enabled on the DVM.
+
+In this example, the user will deposit wETH into the `OptimisticDepositBox` and register it with the `ETH/USD` price identifier.
+
+The user can now withdraw a USD-denominated amount of wETH from their `DepositBox` via smart contract calls. The feature introduced by the Optimistic Oracle is optimistic on-chain pricing of the user's ERC-20 balance.
```

Comment:
"Can now..." - Can now after what? Doesn't seem clear to me what the cause of being able to do this is now.

Reply:
That's a good point. This is kind a copy-paste artifact from the DVM tutorial and I can make it clearer.